cpp/onedal/decomposition/pca/backend/cpu/local_error_convertor.hpp

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+ * Copyright 2020 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#pragma once
+
+#include "onedal/backend/interop/common_error_convertor.hpp"
+
+namespace dal {
+namespace decomposition {
+namespace pca {
+namespace backend {
+
+static void local_status_to_exception(daal::services::Status s) {
+    dal::backend::interop::common_status_to_exception(s);
+}
```

Comment:
I would suggest not to divide error handling to "local" and "common" part like this, but to pass a lambda function in the `common_status_to_exception()`, which will play a role of "local" conversion routine. Is it possible?